traffic_ops/traffic_ops_golang/api/api.go

```diff
@@ -492,6 +492,7 @@ type APIInfo struct {
 	ReqID     uint64
 	Version   *Version
 	Tx        *sqlx.Tx
+	CancelTx  context.CancelFunc
```

Comment:
Shouldn't the `APIInfo.Close()` method cancel the context after attempting to commit the DB transaction?

Reply:
Good idea, making APIInfo.Close() cancel the context after attempting to commit the DB transaction in a6436a5c5a.